src/lib/core/overlay/overlay-ref.ts

```diff
@@ -96,6 +103,16 @@ export class OverlayRef implements PortalHost {
     return this._backdropClick.asObservable();
   }
 
+  /** Returns an observable that emits when the overlay has been attached. */
+  onAttach(): Observable<void> {
+    return this._onAttach.asObservable();
+  }
+
+  /** Returns an observable that emits when the overlay has been detached. */
+  onDetach(): Observable<void> {
+    return this._onDetach.asObservable();
+  }
```

Comment:
Can we make these getters called `attachments` and `detachments`? I kind of want to move away from the `onAction`-style naming for observables